File: engine/engine-model/src/main/scala/tools/packet/cashshop/gachapon/CashShopGachaponFailed.scala
package tools.packet.cashshop.gachapon

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class CashShopGachaponFailed() extends PacketInput {
  override def opcode(): SendOpcode = SendOpcode.CASHSHOP_CASH_ITEM_GACHAPON_RESULT
}

File: engine/engine-model/src/main/java/tools/packet/cashshop/gachapon/CashShopGachaponFailed.java
package tools.packet.cashshop.gachapon;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record CashShopGachaponFailed() implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CASH_SHOP_CASH_ITEM_GACHAPON_RESULT;
   }
}